cmd/node/main.go

```diff
@@ -660,7 +660,10 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		return err
 	}
 
-	metrics.SaveCurrentNodeName(coreComponents.StatusHandler, preferencesConfig.Preferences.NodeDisplayName)
+	metrics.SaveNodeNameChainIdAndMinGasPrice(coreComponents.StatusHandler,
```

Comment:
I don't like this. Think to other approach. Maybe add a function `SaveMetricValue(key string, value interface{})` or 2 separate functions: one for string values and one for numeric values

Reply:
done.